tests/global_config_arm64.py

```diff
@@ -1,20 +1,52 @@
+import os
 from typing import Any
 
 import pytest_testconfig
+from ocp_resources.template import Template
 
 from utilities.constants import (
     ARM_64,
     EXPECTED_CLUSTER_INSTANCE_TYPE_LABELS,
+    FLAVOR_STR,
     PREFERENCE_STR,
     Images,
 )
+from utilities.infra import get_latest_os_dict_list
 
 global config
 global_config = pytest_testconfig.load_python(py_file="tests/global_config.py", encoding="utf-8")
 
 Images.Cirros.RAW_IMG_XZ = "cirros-0.4.0-aarch64-disk.raw.xz"
 EXPECTED_CLUSTER_INSTANCE_TYPE_LABELS[PREFERENCE_STR] = f"rhel.9.{ARM_64}"
 
+IMAGE_NAME_STR = "image_name"
+IMAGE_PATH_STR = "image_path"
```

Comment:
can the common ones goto utilities/constants.py?

Reply:
done